spec/features/02_user_visits_show_page_spec.rb

```diff
@@ -54,7 +53,7 @@
   scenario 'show page has review info' do
     visit figure_path(lincoln)
 
-    expect(page).to have_content 'Aggregate Score: 5'
+    expect(page).to have_content 'Average Rating: 5'
```

Comment:
I would add a more stringent way to test average rating, is this tested at the model level? or perhaps add another rating that is not 5 and make sure the page looks as expected